OneCommunityGlobal/HighestGoodNetworkApp
# Vansh Lint fixes for User Management

# Description
Fixed lint errors for User Management Component (`src/components/UserManagement/**`)

## Related PRS (if any):
N/A

## Main changes explained:
- Removed `src/components/UserManagement/**` from `.eslintignore` and `.prettierignore`
- Modified the code to fit rules defined in ESLint and Prettier

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as admin user
4. go to dashboard→ Other Links→ User Management
5. Verify that all the functionalities behave as before.

## Screenshots or videos of changes:

## Note:
Include the information the reviewers need to know.
